### PR TITLE
Add some missing includes

### PR DESCRIPTION
**Pull request by gfgit**
Hi, this fixes some compile errors on KDE Neon Developer with GCC 11.4
Also fix enum names changes on wlroots 0.18.0

### CODE REVIEW

File: como/input/backend/wlroots/pointer.h [hunk 1]
```diff
@@ -104,6 +109,16 @@ void handle_axis(struct wl_listener* listener, void* data)
 
     auto get_source = [](auto wlr_source) {
         switch (wlr_source) {
+#ifdef WLR_HAVE_WL_POINTER_AXIS_SOURCE
```
[the remark this answers is not in the document]
Comment (gfgit) [reply como/input/backend/wlroots/pointer.h]: Done

**Comment by gfgit** [comment]
> Code changes look good. 👍 There is one more commit message to change.

Which commit?

**Comment by romangg** [comment]
> > Code changes look good. 👍 There is one more commit message to change.
> 
> Which commit?

You can see that usually [in the job output](https://github.com/winft/como/actions/runs/8156516440/job/22297357661#step:4:15) of the failing job.

In this case the commit `input: fix: use new wayland enums` has body lines that are too long. Note that GitHub automatically folds too long lines. But if some dev later on tries to read the log on terminal it doesn't. To ensure readability of the git log, there is a max line length.

**Comment by romangg** [comment]
The clang-format job is failing: https://github.com/winft/como/actions/runs/8159949886/job/22305582476?pr=16

If you use VS Code, then it should automatically detect the clang-format config in IDE. Press Ctrl+Shift+I.

If you use Qt Creator there are few steps to do to setup a key combination: https://www.vikingsoftware.com/blog/using-clang-format-with-qtcreator/

**Comment by gfgit** [comment]
> The clang-format job is failing: https://github.com/winft/como/actions/runs/8159949886/job/22305582476?pr=16
> 
> If you use VS Code, then it should automatically detect the clang-format config in IDE. Press Ctrl+Shift+I.
> 
> If you use Qt Creator there are few steps to do to setup a key combination: https://www.vikingsoftware.com/blog/using-clang-format-with-qtcreator/

Thanks didn't know that!
Though now it's a bit weird with firs argument not indented and others indented because of preprocessor macros

**Comment by romangg** [comment]
Yea, so my philosophy with code style is: better have it sometimes weird but always consistent. 😄

**Pull request merged**